src/scripts/main.js

```diff
@@ -1,3 +1,66 @@
 'use strict';
 
-// write code here
+const field = document.querySelector('.field');
+const buttons = document.querySelectorAll('.button');
+
+buttons.forEach(item => item.addEventListener('click', (e) => {
+  const target = e.target;
+
+  const tBody = field.children[0];
+  const rows = field.children[0].children;
+  const columns = rows[0].children;
+
```

Comment:
you can use native selectors for form elements)
https://learn.javascript.ru/dom-navigation#dom-navigation-tables